Gemfile

```diff
@@ -87,7 +87,7 @@ gem 'pg', '~> 0.18.0', '< 0.19.0'
 gem 'acts_as_versioned', :git => 'https://github.com/technoweenie/acts_as_versioned.git', :ref => '63b1fc8529d028'
 gem 'active_model_otp', :git => 'https://github.com/heapsource/active_model_otp.git', :ref => '55d93a3979'
 gem 'cancancan', '~> 1.12.0', '< 1.13.0'
-gem 'charlock_holmes', '~> 0.7.0', '< 0.7.4'
+gem 'charlock_holmes', '~> 0.7.0'
```

Comment:
I think the full comparison should link to https://github.com/brianmario/charlock_holmes/compare/0.7.3...0.7.5, no?

Looks good otherwise.

Reply:
Yes. I thought I had. Obviously not /o\